crates/dekaf/src/api_client.rs

```diff
@@ -0,0 +1,519 @@
+use anyhow::{anyhow, bail, Context};
+use bytes::{Bytes, BytesMut};
+use deadpool::managed::{self, Manager, Metrics, PoolError, RecycleResult};
```

Comment:
Yup, this is something I wasn't sure about but makes a lot of sense to me. I almost wish there was a `rust-analyzer` feature like `autoimport`, but that automatically qualifies names with their parent module... except then you'd have a bunch of references to e.g `managed::PoolError` which is much less useful than `deadpool::managed:PoolError`, so this might just be more subjective than a rule can define... I'll go through and make this change though 👍